src/screens/CreateEventPage.js

```diff
@@ -100,46 +119,112 @@ class CreateEventPage extends Component {
    contactSubmit(e) {
       e.preventDefault();
       if (this.handleValidation()) {
-         //TODO: this is where the BE content will go
+         let { categoryType, eventNameText, eventDescription, eventLocationText,
+            eventDate, eventStartTime, eventEndTime, eventSpotsAvailable } = this.state.fields
+         console.log(this.state.fields)
+         fetch('http://localhost:8080/events', {
+            method: 'POST',
+            headers: {
+               Accept: 'application/json',
+               'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+               category: categoryType,
+               name: eventNameText,
+               description: eventDescription,
+               location: eventLocationText,
+               date: eventDate,
+               startTime: eventStartTime,
+               endTime: eventEndTime,
+               creator: {
+                  username: "pperez"
+               },
+               availableSpots: eventSpotsAvailable,
+               // availableSpots: eventSpotsAvailable,
+               coordinates: '',
+               public: 'true'
+            }),
+         }).then((response) => response.json())
+            .then((responseJson) => {
+               console.log(responseJson);
+            })
+            .catch((error) => {
+               console.error(error);
+            })
+         alert('Event Created')
+         this.resetFields()
+         console.log(this.state + "reset state")
+
+
       } else {
          //TODO: this is where the BE content will go
```

Comment:
What is this TODO for? If it is something that needs to get done then we should finish it before merging. If it is just a placeholder then either remove the else statement or leave a better description of what needs to be done

Reply:
Or maybe put a ticket number in the TODO. This would require a groomed ticket in Jira, but that way the TODO work is tied to actual work, and we know when it will get done.